src/lavinmqctl.cr

```diff
@@ -380,50 +412,21 @@ class LavinMQCtl
     columns = ARGV
     columns = ["user", "peer_host", "peer_port", "state"] if columns.empty?
     resp = http.get "/api/connections", @headers
-    puts "Listing connections ..." unless quiet?
     handle_response(resp, 200)
     if conns = JSON.parse(resp.body).as_a?
-      puts columns.join("\t")
-      conns.each do |u|
-        if conn = u.as_h?
-          columns.each_with_index do |c, i|
-            case c
-            when "client_properties"
-              # JSON.build(STDOUT) { |json| conn[c].to_json(json) }
-              print_erlang_terms(conn[c].as_h)
-            else
-              print conn[c]
-            end
-            print "\t" unless i == columns.size - 1
-          end
-          puts
-        end
+      cc = conns.map do |u|
+        next unless conn = u.as_h?
+        conn.select { |k, v| columns.includes? k }
       end
+      output cc, columns
     else
       abort "invalid data"
     end
   end
 
-  private def print_erlang_terms(h : Hash)
```

Comment:
We are dropping rabbitmqctl compatible output?

Reply:
No, I'll add it back. Didn't see a reason to keep it, but didn't think about compatibility.